web/src/invite.ts

```diff
@@ -329,6 +343,15 @@ function open_invite_user_modal(e: JQuery.ClickEvent<Document, undefined>): void
         set_expires_on_text();
         set_streams_to_join_list_visibility();
 
+        $("#invite-user-modal").on("click", ".setup-tips-container .banner_content a", () => {
+            dialog_widget.close();
+        });
+
+        $("#invite-user-modal").on("click", `.main-view-banner-close-button`, (event) => {
```

Comment:
Why `main-view-banner-close-button` is enclosed in backticks here and not inside quotes like we do elsewhere?

Reply:
Fixed and my bad for overlooking this. Thanks!